science/limesuite/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+name                limesuite
+version             v19.01.0
+categories          science comms
+platforms           darwin macosx
+license             Apache-2
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
```

Comment:
add me here & I'll help maintain this port, as I do with most of the SDR-oriented ports.

Reply:
added